iocore/cache/I_Cache.h

```diff
@@ -73,7 +73,7 @@ struct CacheProcessor : public Processor {
   {
   }
 
-  virtual int start(int n_cache_threads = 0, size_t stacksize = DEFAULT_STACKSIZE);
+  virtual int start(int n_cache_threads = 0, size_t stacksize = DEFAULT_STACK_SIZE) override;
```

Comment:
How come the name of the constant is changed to add an underscore but the variable name is not?